media/src/iosMain/kotlin/dev/icerock/moko/media/picker/ios/MediaPickerController.kt

```diff
@@ -65,7 +68,21 @@ class MediaPickerController(
     }
 
     override suspend fun pickFiles(): FileMedia {
-        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+        var delegatePtr: DocumentPickerDelegateToContinuation? // strong reference to delegate (view controller have weak ref)
+        val fileMedia = suspendCoroutine<FileMedia> { continuation ->
+            val localDelegatePtr = DocumentPickerDelegateToContinuation(continuation)
+            delegatePtr = localDelegatePtr
+
+            val controller = UIDocumentPickerViewController(forOpeningContentTypes = listOf(UTType.typeWithIdentifier("public.data")))
```

Comment:
what this type means? 
also can we use constant for type identifier? as like `kUTTypeVideo` above.
also maybe we should pass type as argument? or this type means all file types at all?

Reply:
public.data is a parent for all standard types https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/understanding_utis/art/conformance_hierarchy.gif